org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/dom/SchemaLocation.java

```diff
@@ -12,40 +12,72 @@
  */
 package org.eclipse.lemminx.dom;
 
+import java.util.Collection;
 import java.util.HashMap;
 import java.util.Map;
-import java.util.StringTokenizer;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 /**
- * 
+ *
  * The declared "xsi:schemaLocation"
  */
 public class SchemaLocation {
 
-	private final Map<String, String> schemaLocationValuePairs;
+	private final Map<String, SchemaLocationHint> schemaLocationValuePairs;
 
 	private final DOMAttr attr;
 
-	public SchemaLocation(String base, DOMAttr attr) {
+	// The text to match is of the form:
+	// http://example.org root.xsd http://example.org bison.xsd http://example.org potato.xsd

```

Comment:
This is not a valid `schemaLocation` since the namespace URI are the same (same namespace is not allowed)

Reply:
Okay. I resolved this and put in a proper example.